Makefile

```
@@ -0,0 +1,68 @@
+.DEFAULT_GOAL := help
+COMPOSE := docker compose
+
+.PHONY: help up down dev build rebuild logs clean test test-env test-env-down lint env
+
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (19).

(maxbodylength)